powerVCF.psm1

```diff
@@ -3001,6 +3001,78 @@ Function Get-VCFvRSLCMEnvironment {
 }
 Export-ModuleMember -Function Get-VCFvRSLCMEnvironment
 
+Function New-VCFvRSLCM {
+<#
+    .SYNOPSIS
+    Deploys vRealize Suite Lifecycle Manager
+    
+    .DESCRIPTION
+    Deploys vRealize Suite Lifecycle Manager to the specified network.
+    
+    .EXAMPLE
+    PS C:\> New-VCFvRSLCM -json .\SampleJson\vRealize\New-vRSLCM.json
+    This example Deploys vRealize Suite Lifecycle Manager using a supplied json file
+    
+    #>
+
+    param (
+        [Parameter (Mandatory=$true)]
+            [ValidateNotNullOrEmpty()]
+            [string]$json
+    )
+
+    if (!(Test-Path $json)) {
+        Throw "JSON File Not Found"
+    }
+    else {
+        try {
```

Comment:
Capital 'T'
Getting picky now but I spent a lot of time making sure the code was consistent.